### PR TITLE
Add section for account renewal

### PR DESCRIPTION
**Pull request by gandatchabana**
Links to DRAC FAQ and support emails

resolve: #179

### CODE REVIEW

File: docs/Extra_compute_drac.rst [hunk 1]
```diff
@@ -103,6 +103,18 @@ your supervisor is member of the ``rrg-bengioy-ad`` allocation, you should apply
...
+---------------
+
+All user accounts (Sponsor & Sponsored) have to be renewed annually in order to keep up-to-date information on active accounts
+and to deactivate unusued accounts.
+
+To find out how to renew your account or for any other questions regarding DRAC's accounts renewal,
+please head over to their `FAQ <https://alliancecan.ca/en/services/advanced-research-computing/account-management/account-renewals/account-renewals-faq>`_.
+
+If the FAQ is of no help, you can contact DRAC renewal support team at ``renewals@tech.alliancecan.ca``
+or the general support team at ``support@tech.alliancecan.ca``.
```
Comment (satyaog) [inline docs/Extra_compute_drac.rst:116]: 80 chars line width

```suggestion
All user accounts (Sponsor & Sponsored) have to be renewed annually in order to
keep up-to-date information on active accounts and to deactivate unusued
accounts.

To find out how to renew your account or for any other questions regarding
DRAC's accounts renewal, please head over to their `FAQ
<https://alliancecan.ca/en/services/advanced-research-computing/account-management/account-renewals/account-renewals-faq>`_.

If the FAQ is of no help, you can contact DRAC renewal support team at
``renewals@tech.alliancecan.ca`` or the general support team at
``support@tech.alliancecan.ca``.
```

**Review by satyaog — APPROVED** [review]

File: docs/Extra_compute_drac.rst [hunk 2]
```diff
@@ -103,6 +103,21 @@ your supervisor is member of the ``rrg-bengioy-ad`` allocation, you should apply
 you should apply separately using your supervisor's CCRI to have access to ``def-<yoursupervisor>``. Ask your supervisor
 to share these CCRI with you.
 
+Account Renewal
+---------------
+
+All user accounts (Sponsor & Sponsored) have to be renewed annually in order to
+keep up-to-date information on active accounts and to deactivate unusued
```
Comment (btravouillon) [inline docs/Extra_compute_drac.rst:110]:
```suggestion
keep up-to-date information on active accounts and to deactivate unused
```

File: docs/Extra_compute_drac.rst [hunk 3]
```diff
@@ -103,6 +103,21 @@ your supervisor is member of the ``rrg-bengioy-ad`` allocation, you should apply
...
+accounts.
+
+To find out how to renew your account or for any other questions regarding
```
Comment (btravouillon) [inline docs/Extra_compute_drac.rst:113]:
```suggestion
To find out how to renew your account or for any other question regarding
```

**Review by btravouillon — CHANGES_REQUESTED** [review]

**Comment by satyaog** [comment]
@gandatchabana would you have some time to update this PR?

**Pull request merged**